### ISSUE TITLE
Creating user table fails

### ISSUE THREAD

**Issue by pp23** [iljabauer/quamo-wiki#34]
During tests with the db the creation of the user table failed. I'm not sure whether the tablename "user" is recognized as keyword? Postgres throws the following error:

Unsuccessful: create table user (id int4 not null, admin boolean not null, fullname varchar(255) not null, password varchar(255) not null, username varchar(255) not null, setting int4, primary key (id))
10:50:33.527 [main] ERROR o.h.tool.hbm2ddl.SchemaExport - ERROR: syntax error at or near "user"

**Comment by iljabauer** [comment]
Yeah user is a reserved word in postgres. What would you expect HEDL to do? Should it warn you? Just for the bug report I will create
